### DISCUSSION TITLE
Updating to latest RC

### DISCUSSION THREAD

**Discussion by BBaFSE** [danny-avila/LibreChat#4913] [Troubleshooting] [answered]
Hello all,

Thanks for a great product! 

I am following [instructions here ](https://www.librechat.ai/docs/remote/docker_linux#part-iii-updating-librechat) but I can't seem to update to 0.7.6-rc1, it's still showing 0.7.5. I'm running Docker remote hosting on a Hetzner VPS.

Am I missing something?

Thanks!

**Comment by danny-avila** [comment, marked as answer]
The update did not include v0.7.6-rc1 in the footer/server startup as usual, because I aim to do a full release sooner rather than later (wtihin the week)

**Reply by BBaFSE** [reply]
Perfect! Thank you so much for all the hard work.
